aws/resource_aws_security_group_test.go

```diff
@@ -1648,7 +1632,11 @@ func TestAccAWSSecurityGroup_driftComplex(t *testing.T) {
 			{
 				ResourceName:     resourceName,
 				ImportState:      true,
-				ImportStateCheck: testAccAWSSecurityGroupImportStateCheckFunc(5),
+				ImportStateCheck: testAccAWSSecurityGroupImportStateCheckFunc(1),
+				// Below checks commented out, but will check state count.
+				// In rules with cidr_block drift, import only creates a single ingress
+				// rule with the cidr_blocks de-normalized. During subsequent apply, its
+				// normalized to create the 2 ingress rules seen in checks above.
 				//ImportStateVerify:       true,
 				//ImportStateVerifyIgnore: []string{"revoke_rules_on_delete"},
```

Comment:
In cases like these, we can remove the state count check and uncomment the `ImportStateVerify` and `ImportStateVerifyIgnore` fields and but also add the `ingress` and `egress` attributes to this string array as we expect there to be a non-empty plan. I'd leave the comments you have though to describe this expectation in behavior.